back/app/models/user.rb

```diff
@@ -202,6 +202,7 @@ def oldest_admin
     where.not(id: project_folder_moderator(*project_folder_ids))
   }
 
+  scope :admin_or_moderator, -> { where(id: admin).or(where(id: project_moderator)).or(where(id: project_folder_moderator)) }
```

Comment:
Think this should go in the `UserRoles` concern that Alex recently split out

Reply:
However, where are you using this? Doesn't seem to be used anywhere in your PR

Reply:
Very good question! I guess I was using it, then didn't need it. Will remove.